demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -158,6 +163,10 @@ def main():
         log.warning('The "--anchors" options works only for "-at==yolov4". Option will be omitted')
     if args.architecture_type != 'yolov4' and args.masks:
         log.warning('The "--masks" options works only for "-at==yolov4". Option will be omitted')
+    if args.architecture_type != 'nanodet' and args.is_legacy:
+        log.warning('The "--is_legacy" options works only for "-at==nanodet". Option will be omitted')
```

Comment:
Please, could you apply it for all log.warning messages? 
```suggestion
        log.warning('The "--is_legacy" option works only for "-at==nanodet". Option will be omitted')
```

Reply:
Done